learning_resources_search/tasks.py

```diff
@@ -660,6 +689,81 @@ def start_recreate_index(self, indexes, remove_existing_reindexing_tags):
     )
 
 
+@app.task(bind=True)
+def start_embed_resources(self, indexes, skip_content_files):
+    index_tasks = []
```

Comment:
Might be worth checking here to see if the required `QDRANT_*` settings have values, and if not return immediately.

Reply:
Added a check for this